specifyweb/frontend/js_src/lib/components/Attachments/Viewer.tsx

```diff
@@ -46,6 +47,12 @@ export function AttachmentViewer({
     false
   );
 
+  const [displayAttachment] = userPreferences.use(
+    'attachments',
+    'behavior',
+    'display'
```

Comment:
rename to `displayOriginal` or something more helpful like that

Reply:
I meant replace `'display'` user pref name with `'displayOriginal'`

and the variable name should stay `displayOriginal` like it is now